modules/oit/src/processors/rasterizationrenderer.cpp

```diff
@@ -240,6 +240,13 @@ void RasterizationRenderer::process() {
     }
 
     utilgl::deactivateCurrentTarget();
+
+    // Set rasterization processors to valid now that rasterization is done and properties/inports
+    // were used.
+    // @see Rasterization::setValid()
+    for (auto rasterization : rasterizations_) {
+        rasterization->getProcessor()->Processor::setValid();

```

Comment:
This should work, But I think adding a new function like "setValidDelayed" in the rasterizer base, and calling that might be a bit clearer.

Reply:
Good idea. Was even considering that.